languages/python/concepts/bools/links.json

```diff
@@ -1 +1,22 @@
-[]
+[
+  {
+    "url": "https://docs.python.org/3/library/stdtypes.html#truth",
+    "description": "boolean"
+  },
+  {
+    "url": "https://docs.python.org/3/library/stdtypes.html#truth",
+    "description": "boolean"
+  },
```

Comment:
There are duplicate entries in this file; is this intentional? I'm guessing it's a result of the script used counting reference-style links as separate links... but it was still able to grab the referenced URL though?

@ErikSchierboom, can you clarify the intent here?

Reply:
The script was basically just a dumb scraper. Duplicates should definitely be removed.